### PR TITLE
fix: Wrap await in async function

### PR DESCRIPTION
**Pull request by dominicwong617**
## Description

Wrap await in async function in "Step 3: Reviewing account info"

1. Motivation for change?
2. What was changed?
3. Link to relevant issues?

## Checklist

- [x] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] New links to files and images were verified
- [x] For fixes/refactors: all existing references were identified and replaced
- [x] [Style guide](https://developers.google.com/style) was reviewed and applied
- [x] Clear code samples were provided
- [x] People were tagged for review

### CODE REVIEW

**Comment by agraebe** [comment]
thanks for submitting this PR. This is an interesting one because we'd need to agree on a convention. If we do this here, we should do this in all other places - which reasons me to believe we shouldn't introduce this.

A developer should be able to put this into a method and run it in a way that makes the most sense. 

Happy to hear what y'all thinking

**Comment by dominicwong617** [comment]
When I did the tutorial, I didn't imagine the code sample to not work as my focus was to learn your libraries. It would be better if developers can focus on learning BlockStack instead of fixing JavaScript code (It might also not be obvious for non-JS developers). I am happy to change the other occurrences as well if needed.

**Comment by agraebe** [comment]
I get the point. I think non-JS devs would be challenged in a variety of places and we'd need to find the right "baseline" for expectations. Without going into all that, a quick fix might be to just wrap the code pieces into methods that could be copy&pasted?

For instance:
```js
async function getAccountInfo() {
  const accounts = new AccountsApi(apiConfig);

  const accountInfo = await accounts.getAccountInfo({
    principal: stacksAddress
  });

  return accountInfo;
}
```

wdyt? can we update the PR to do that?

**Comment by dominicwong617** [comment]
Updated!

File: src/pages/understand-stacks/managing-accounts.md [hunk 1]
```diff
@@ -75,15 +75,15 @@ const stacksAddress = getAddressFromPrivateKey(
   TransactionVersion.Testnet // remove for Mainnet addresses
 );
 
-...
-
+async function getAccountInfo() {
   const accounts = new AccountsApi(apiConfig);
```
Comment (agraebe) [inline src/pages/understand-stacks/managing-accounts.md:79]: I think this was is used in other methods, I'd suggest initializing it outside of this method. so the other examples could work with copy&paste

File: src/pages/understand-stacks/managing-accounts.md [hunk 2]
```diff
@@ -106,24 +106,30 @@ The `balance` property represents the Stacks token balance, as hex-encoded strin
 Proofs, provided as hex-encoded strings, can be removed from the responses by setting the `proof` parameter:
 
 ```js
-const accountInfo = await accounts.getAccountInfo({
-  principal: stacksAddress,
-  proof: 0,
-});
+async function getAccountInfo() {
```
Comment (agraebe) [inline src/pages/understand-stacks/managing-accounts.md:109]: I would rename this method so it is not the same as the one above, eg `getAccountInfoWithoutProof`

**Review by agraebe — COMMENTED** [review]
I think this would be a good change - left 2 comments. thanks 🙏🏼

File: src/pages/understand-stacks/managing-accounts.md (hunk 1 above)
Comment (dominicwong617) [reply src/pages/understand-stacks/managing-accounts.md]: Fixed

File: src/pages/understand-stacks/managing-accounts.md (hunk 2 above)
Comment (dominicwong617) [reply src/pages/understand-stacks/managing-accounts.md]: Fixed

**Comment by dominicwong617** [comment]
Updated!

**Review by agraebe — APPROVED** [review]
Thanks for the contribution! 👍

**Pull request merged**